src/gui/properties/peerlistwidget.cpp

```diff
@@ -451,7 +451,14 @@ void PeerListWidget::wheelEvent(QWheelEvent *event)
         // Shift + scroll = horizontal scroll
         event->accept();
 
-        QWheelEvent scrollHEvent(event->pos(), event->globalPos(), event->delta(), event->buttons(), event->modifiers(), Qt::Horizontal);
+#if (QT_VERSION >= QT_VERSION_CHECK(5, 14, 0))
+        QWheelEvent scrollHEvent(event->position(), event->globalPosition()
```

Comment:
Do you know what's the point in creating this variable?

Reply:
Isn't it for `QTreeView::wheelEvent(&scrollHEvent);`? Also there is this part: `event->angleDelta().transposed()`.

Reply:
Well, seems it was to perform horizontal scrolling when Shift modifier is used. Does new code branch behave the same way?

Reply:
>Does new code branch behave the same way?

Yes, I tested the code.

Reply:
I just don't understand... old code has `Qt::Horizontal` explicitly set but the new one has no such flag. It's unclear for me.

Reply:
QWheelEvent has only one non-deprecated constructor now: https://doc.qt.io/qt-5/qwheelevent.html#QWheelEvent-6
All other constructors are deprecated.

Reply:
Your new logic is still different from the old one (although it's not a problem for me). Old code performs horizontal scrolling when Shift is hold (independent from original direction). New code performs inverted scrolling in such case (so if user scrolls horizontally with Shift it will be inverted to vertical scrolling).

Reply:
>New code performs inverted scrolling in such case (so if user scrolls horizontally with Shift it will be inverted to vertical scrolling).

I think anyone who is able to scroll horizontally (for example a touchpad) won't bother to use the Shift modifier + vertical scrolling and I doubt they will use the Shift modifier at all.
Of course I will fix it properly if this is really a problem for some users.